README.md

```diff
@@ -1,39 +1,90 @@
 # Numeric Separators
 
-This is a [strawman proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
+## Stage 1
+
+This is a [proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
```

Comment:
Ugh, I may have accepted a PR that will conflict with this. Could you please merge and resolve the conflict?